app/models/bookings/booking.rb

```diff
@@ -52,10 +52,11 @@ class Booking < ApplicationRecord
     UPCOMING_TIMEFRAME = 2.weeks
```

Comment:
Can `UPCOMING_TIMEFRAME`  be removed too?

Reply:
Good catch!